pkg/volume/plugins.go

```diff
@@ -787,6 +787,9 @@ func (pm *VolumePluginMgr) refreshProbedPlugins() {
 
 // ListVolumePluginWithLimits returns plugins that have volume limits on nodes
 func (pm *VolumePluginMgr) ListVolumePluginWithLimits() []VolumePluginWithAttachLimits {
+	pm.mutex.Lock()
```

Comment:
Since its only reading from the map and not modifying it, why not use a **RLock()** instead ?